packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1085,6 +1085,8 @@ export class AccountHandler {
         (this.config.securityHistory.ipProfiling
           .allowedRecency as unknown as number) || 0;
       if (securityEventVerified && securityEventRecency < allowedRecency) {
+        this.glean.login.fromKnownIp(request);
+        this.statsd.increment('account.ipprofiling.seenAddress');
```

Comment:
Nice, maybe this can be `account.signin.confirm.bypass.ip` to keep all the bypass metrics together?